qb/config.libs.sh

```diff
@@ -509,6 +509,13 @@ if [ "$HAVE_MENU" != 'no' ]; then
    fi
 fi
 
+if [ "$HAVE_MENU_WIDGETS" != 'no' ]; then
+   die : 'Notice: Menu widgets are not fully implemented and should not be enabled'
+   die : 'Please do not report any bug concerning widgets until this message is removed'
+   die : 'Press any key to continue'
+   read widgets_prompt
```

Comment:
I'm not sure its a good idea to make the configure script require input to continue, a warning is enough.

Also you only need to call the `die` function once, every argument will be printed on a new line, for example.
```
die : 'Notice: Menu widgets are not fully implemented and should not be enabled' \
   'Please do not report any bug concerning widgets until this message is removed'
```